examples/drawing/plot_simple_graph.py

```diff
@@ -0,0 +1,38 @@
+"""
+============
+Simple graph
+============
+
+Draw simple graph with manual layout.
+"""
+
+import networkx as nx
+import matplotlib.pyplot as plt
+
+G = nx.Graph()
+G.add_edge(1, 2)
+G.add_edge(1, 3)
+G.add_edge(1, 5)
+G.add_edge(2, 3)
+G.add_edge(3, 4)
+G.add_edge(4, 5)
+
+# explicitly set positions
+pos = {1: (0, 0), 2: (-0.1, 0.03), 3: (0.2, 0.017), 4: (0.4, 0.0255), 5: (0.5, 0.003)}
```

Comment:
This is a very good example to have in the Gallery.  
Just the kind of thing someone would want to do and not know how to do it.

Perhaps they wouldn't want so many decimals in their positions though...
Would they more naturally use integers (or half integers).  
I usually use manually entered values when I want a grid of nodes... Like what is used for the dedensify example.
For example (integers for left columns then need 0.5 to center the other columns vertically)

```python
G=nx.DiGraph([(0, 3), (1, 3), (2, 4), (3, 5), (3, 6), (4, 6), (5, 6)])

# group nodes by column
left_nodes = [0, 1, 2]
middle_nodes = [3, 4]
right_nodes = [5, 6]

# set the position according to column (x-coord)
pos = {n: (0, i) for i, n in enumerate(left_nodes)}
pos.update({n: (1, i + 0.5) for i, n in enumerate(middle_nodes)})
pos.update({n: (2, i + 0.5) for i, n in enumerate(right_nodes)})
```

Not a big deal, and maybe it doesn't look as nice.  I haven't tried drawing it.